lib/response.js

```diff
@@ -40,6 +40,7 @@ class Fido2Result {
 		await this.validateTokenBinding();
 		await this.validateId();
 
+
```

Comment:
Not sure why you changed formatting here

Reply:
I have removed the empty line.